testing/test_aux.out

```diff
@@ -30,7 +30,7 @@ RESULT skip security.passwords.telnet Could not lookup password info for mac-key
 RESULT skip security.passwords.ssh Could not lookup password info for mac-key 9a:02:57:1e:8f:01
 RESULT fail security.passwords.http Default passwords has not been changed
 RESULT fail security.passwords.https Default passwords has not been changed
-RESULT fail security.passwords.telnet Default passwords has not been changed
+RESULT pass security.passwords.telnet  Default passwords have been changed
```

Comment:
that extra space is killing me...

Reply:
Oh yes, and the wrong plural noun and singular person verb is killing me too!
I'll sort them out in another PR about the passwords tests.

Reply:
I really need to sort out the password tests, they keep on changing results all the time!